rotkehlchen/tests/unit/test_cost_basis.py

```diff
@@ -580,3 +584,59 @@ def test_missing_acquisitions(accountant):
         time=4,
     ))
     assert cost_basis.missing_acquisitions == expected_missing_acquisitions
+
+
+@pytest.mark.parametrize('mocked_price_queries', [{
+    A_ETH: {A_EUR: {1469020840: ONE}},
+    A_3CRV: {A_EUR: {1469020840: ONE}},
+}])
+def test_swaps_acquisitions(accountant):
+    """Check acquisition part of swaps doesn't count as taxable."""
+    pot = accountant.pots[0]
+    transactions_accountant = pot.transactions
+    transactions_accountant._process_tx_swap(
+        timestamp=1469020840,
+        out_event=HistoryBaseEntry(
+            event_identifier=b'123',
+            sequence_index=1,
+            timestamp=Timestamp(1469020840),
+            location=Location.BLOCKCHAIN,
+            location_label='location-label',
+            asset=A_ETH,
+            balance=Balance(amount=ONE, usd_value=ONE),
+            notes='',
```

Comment:
this is probably the least relevant field since we don't check it at all. But would be nice to write something close to what an amm swap would have